### DISCUSSION TITLE
Patch-Release 4.0.2

### DISCUSSION THREAD

**Discussion by cz-dev-ge** [nunit/nunit#4624] [Q&A] [answered]
Hi there,
we are waiting for a bugfix that has been merged after the 4.0.1 release. What is your release schedule for the 4.0.2? 

Best regards,
Georg

**Comment by OsirisTerje** [comment]
Which bug-fix is that?

**Comment by cz-dev-ge** [comment]
#4608

**Comment by OsirisTerje** [comment]
They are both set to milestone 4.1 and the PRs are part of that release train already.  

Anyway, I think it is due to get that out anyway real soon now, so I'll put a date on it and we can start checking if any other bugs needs to be in there.  The list of open issues for the  4.1. does not contain any more bugs.

@stevenaw @manfred-brands Comments ?

**Comment by cz-dev-ge** [comment]
Alright. Thanks for your quick reply.

**Comment by manfred-brands** [comment]
@OsirisTerje I was working on #4537 and ran into some issue, but I don't think it is that urgent.

**Comment by OsirisTerje** [comment]
4.1 is scheduled for next Sunday, the 25th

**Comment by stevenaw** [comment]
@OsirisTerje Nothing else on my end that "must" get done.

As a nice-to-have, I'll be trying to wrap up some changes to the PR for https://github.com/nunit/nunit/issues/1215 this weekend
https://github.com/nunit/nunit/issues/3981 is then next on my list but that also doesn't "have" to be done in advance of a given release.

**Comment by OsirisTerje** [comment, marked as answer]
@georg-eckert-zeiss Can you please check the package https://www.myget.org/feed/nunit/package/nuget/NUnit/4.1.0-alpha.0.1

**Reply by cz-dev-ge** [reply]
Hi @OsirisTerje - sorry for the late reply. I was out-of-office. I noticed you released 4.1.0 and I guess you don't need me to check it anymore, right?

**Reply by OsirisTerje** [reply]
As long as you're happy with it, we're happy :-)

**Reply by cz-dev-ge** [reply]
All fine. Thanks.
